pkgs/tools/backup/gh2md/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+, requests
+, python-dateutil
+}:
+
+buildPythonPackage rec {
```

```suggestion
buildPythonApplication rec {
```

it's outside of python-modules